mozilla-release/browser/base/content/browser.js

```diff
@@ -858,9 +924,9 @@ function _loadURIWithFlags(browser, uri, params) {
         browser.webNavigation.setOriginAttributesBeforeLoading({ userContextId: params.userContextId });
       }
 
-      browser.webNavigation.loadURIWithOptions(uri, flags,
-          referrer, referrerPolicy, postData, null, null,
-          !!params.ensurePrivate);
+      browser.webNavigation.loadURIWithOptions(uri, flags, referrer, referrerPolicy,
```

Comment:
It shows this error `NS_ERROR_MALFORMED_URI: Component returned failure code: 0x804b000a (NS_ERROR_MALFORMED_URI) [nsIWebNavigation.loadURIWithOptions]`
when opening the about:addons page